js/adapt-youtube.js

```diff
@@ -44,19 +42,18 @@ define([
 
       $iframe.width(widgetWidth);
 
-      var aspectRatio = (this.model.get('_media')._aspectRatio ? parseFloat(this.model.get('_media')._aspectRatio) : 1.778);//default to 16:9 if not specified
-      if (!isNaN(aspectRatio)) {
-        $iframe.height(widgetWidth / aspectRatio);
-      }
+      var aspectRatio = (this.model.get('_media')._aspectRatio ? parseFloat(this.model.get('_media')._aspectRatio) : 1.778);// default to 16:9 if not specified
+      if (isNaN(aspectRatio)) return;
+
```

Comment:
is this empty line needed? Also I would place this comment `// default to 16:9 if not specified` above where the aspectRatio variable is initialised.